content/applications/services/helpdesk.rst

```diff
@@ -32,7 +31,6 @@ team is assigned to, select it from the :guilabel:`Company` drop-down menu.
    use only.
 
    .. image:: helpdesk/team-description-webform.png
-      :align: center
       :alt: View of a Helpdesk team's website form displaying the team description.
 
 Visibility & Assignment
```

Comment:
For the warning block starting in line 64, I would revise to be more explicit about field changes:

.. warning::
   A team's visibility can be altered after the initial configuration. However, if the team changes
   from public ('Invited portal users and all internal users') access to either private ('Invited internal users') or company- only ('All internal users') access, portal users are removed as followers from both the team, and from individual tickets.